pkg/controller/podautoscaler/metrics/metrics_client_test.go

```diff
@@ -365,54 +266,46 @@ func TestQpsSumEqualZero(t *testing.T) {
 
 func TestCPUMoreMetrics(t *testing.T) {
 	tc := testCase{
-		replicas:           5,
-		desiredValue:       5000,
-		desiredRunningPods: 5,
-		targetResource:     "cpu-usage",
+		replicas: 5,
+		desiredResourceValues: PodResourceInfo{
+			"test-pod-0": 5000, "test-pod-1": 5000, "test-pod-2": 5000,
+			"test-pod-3": 5000, "test-pod-4": 5000,
```

Comment:
Shouldn't we have here avg instead of sum? We are calculating avg [here](https://github.com/kubernetes/kubernetes/pull/33593/files#diff-1458692d0adfda08bd7a42d8c9ac46a2R225). Is this test passing?